src/components/base/base-link/BaseLink.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <component
+    :is="elementType"
+    data-testID="ui-link"
+    :id="id"
+    :class="[
+      'base-link',
+      `base-link--is-${type}`,
+      `base-link--is-${size}`,
+      { [`base-link--is-${type}-ALT`]: variant && !disabled },
+      { [`base-link--is-${type}-ALT-disabled`]: variant && disabled },
```

Comment:
yo creo que nos podríamos ahorrar esta clase `base-link--is-${type}-ALT-disabled` y solo utilizar el estado disabled para aplicar los estilos

Reply:
Se realizo el cambio y se maneja por el estado